components/rating/rating.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <title>Rating | Components</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <a class="navbar-brand" href="#">
+            <p class="brand-head wh-text">COMP-UI</p>
+        </a>
+        <div class="collapse navbar-collapse" id="collapsibleNavbar">
+            <ul class="navbar-nav ml-auto">
+                <li class="navbar-item">
+                    <a class="nav-link active wh-text" href="index.html">Home</a>
+                </li>
+                <li class="navbar-item">
+                    <a class="nav-link wh-text" href="index.html">Docs</a>
```

Comment:
Path for docs is index.html , just check if thats the path for docs you want

Reply:
it's on the same page actually, Probably you could understand after I put the landing page live.
Thanks